content/cybersecurity/concepts/malware/terms/trojan/trojan.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'Trojan'
+Description: 'Trojan is a form of downloaded malware that disguises itself as a legitimate application so it can trick users into downloading it, thus taking control of over their computer and other devices'
+Subjects:
+  - 'Cybersecurity'
+  - 'Computer Science'
+Tags:
+  - 'Cybersecurity'
+  - 'Malware'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+Like [the war tactic](https://www.history.com/topics/ancient-greece/trojan-war) used by the ancient Greeks in their fight against Troy, **Trojan** is a malware pretends to be a legitimate application, like an online game, so it can gain control over a user's computer and other devices.
```

Comment:
```suggestion
Like [the war tactic](https://www.history.com/topics/ancient-greece/trojan-war) used by the ancient Greeks in their fight against Troy, A **Trojan** is a malware pretends to be a legitimate application, like an online game, so it can gain control over a user's computer and other devices.
```